File: striver/sorting/insertionSort.ts
const insertionSort = (arr: number[]) => {
  let n = arr.length;
  for (let i = 0; i < arr.length; i++) {
    let j = i;
    while (j > 0 && arr[j - 1] > arr[j]) {
      let temp = arr[j - 1];
      arr[j - 1] = arr[j];
      arr[j - 1] = temp;
      j--;
    }
  }
  console.log(arr);
};

insertionSort([5, 4, 1, 2, 3]);

File: src/striver/sorting/insertion-sort.py
class Solution:
    def insertion_sort(self,arr,n):
        for j in range(0,n):
            j = i
            while(j > 0 and arr[j-1] > arr[j]):
                arr[j-1],arr[j] = arr[j], arr[j-1]
                j=j-1
